packages/matter-node.js/src/examples/BridgedDevicesNode.ts

```diff
@@ -72,7 +72,7 @@ class BridgedDevice {
         // product name / id and vendor id should match what is in the device certificate
         const vendorId = new VendorId(getIntParameter("vendorid") ?? deviceStorage.get("vendorid", 0xFFF1));
         const productName = `node-matter OnOff-Bridge`;
-        const productId = getIntParameter("productid") ?? deviceStorage.get("productid", 0x8000);
+        const productId = getIntParameter("productid") ?? deviceStorage.get("productid", 0x1000);
```

Comment:
Please do not change default here because this id is officially not allowed for dev devices.

Reply:
it was for testing google project, I'll do it.